alchemist-grid/src/main/java/it/unibo/alchemist/grid/config/LightInfoGeneralSimulationConfig.java

```diff
@@ -25,7 +26,7 @@ public abstract class LightInfoGeneralSimulationConfig implements GeneralSimulat
     private static final long serialVersionUID = 1L;
     private final long endStep;
     private final Time endTime;
-    private final Loader loader;
+    private transient final Loader loader;
```

Comment:
What happens on deserialization?

Reply:
The problem was in serialization, some of the loader contents do not implement Serializable (`org.danilopianini.jirf.FactoryImpl`, `org.jgrapht.graph.DefaultDirectedGraph`, etc.)